browser/resources/brave_webtorrent/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+import("//tools/grit/grit_rule.gni")
```

Comment:
I think this should all move into `components/webtorrent` and resources should be kept inside the component like sync

Reply:
addressed by 32c0fe2